### ISSUE TITLE
TODOs & HOWTOs

### ISSUE THREAD

**Issue by zezhishao** [zezhishao/BasicTS#94]
## How-Tos
- [ ] How to **add new datasets**.
- [ ] How to **config the config file**.
- [ ] How to **inference and save the inference results**.
- [ ] How to **customize loss function**.

## ToDos
Add more baselines:
- [ ] Add DeepAR
- [ ] Add LightGBM
- [x] Add NHiTS
- [ ] Add GRU
- [x] Add STGODE
- [ ] Add SCINet
- [x] Add STSGCN❌, STFGCN❌ (They are built on MXNet)
- [ ] Add SCINet
- [ ] Add TimesNet
- [ ] Add Vanilla Transformer
- [ ] Add GPTs for Time Series

Add more datasets:
- [ ] Add M4, M5, and M6 datasets.

Others:
- [ ] Add AutoML.

**Comment by zezhishao** [comment]
## TODOs
Add more baselines:
- [ ] Add DeepAR
- [ ] Add LightGBM
- [ ] Add NHiTS
- [ ] Add GRU
- [ ] Add STGODE
- [ ] Add SCINet
- [ ] Add STSGCN❌, STFGCN❌ (They are built on MXNet)
- [ ] Add SCINet
- [ ] Add TimesNet
- [ ] Add Vanilla Transformer
- [ ] Add GPTs for Time Series

Add more datasets:
- [ ] Add M4, M5, and M6 datasets.
